prow/kube/client.go

```diff
@@ -386,6 +386,10 @@ func NewClient(c *Cluster, namespace string) (*Client, error) {
 	}, nil
 }
 
+func (c *Client) GetNamespace() string {
```

Comment:
This function doesn't appear to be used.

Reply:
removed